TODO.txt

```diff
@@ -37,6 +37,17 @@ Version 0.18
   ``neighbors`` and update the tests. Set the default value of ``connectivity``
   to 2.
 * In ``skimage.features.corner_peaks``, remove the warning.
+    * In the following docstrings, remove the explicit setting of ``threshold_rel=0``
+        * ``skimage/feature/brief.py::BRIEF``
+        * `` skimage/feature/corner.py::corner_harris``
+        * `` skimage/feature/corner.py::corner_shi_tomasi``
+        * `` skimage/feature/corner.py::corner_foerstner``
+        * `` skimage/feature/corner.py::corner_fast``
+        * `` skimage/feature/corner.py::corner_subpix``
+        * `` skimage/feature/corner.py::corner_peaks``
+        * `` skimage/feature/corner.py::corner_orientations``
+    * In ``skimage/feature/orb.py::_detect_octage`` remove explicitely setting
+      ``threshold_rel=0``.
```

Comment:
Is it really necessary to remove these explicit settings?

Reply:
It's not. But I decided to migrate to the new behavior and they are polluting the codebase ATM